addons/hr_timesheet/models/hr_timesheet.py

```diff
@@ -125,7 +125,7 @@ def _compute_department_id(self):
     @api.model_create_multi
     def create(self, vals_list):
         # Before creating a timesheet, we need to put a valid employee_id in the vals
-        default_user_id = self._default_user()
+        default_user_id = self.env.context.get('user_id', self.env.user.id)
```

Comment:
Isn't it what `self._default_user()` returns? Why changing it?

Reply:
Doesn't exist anymore